### PR TITLE
Issue 687 - add more fission test results

### PR DESCRIPTION
**Pull request by airimovici**

### CODE REVIEW

**Comment by airimovici** [comment]
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/issues/687

File: src/fission/config.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ const PERFORMANCE_COMBO_TABLE = {
     ['start-up-(sessionrestore)', 'Start-up (sessionrestore)', 'sessionrestore'],
     ['start-up-(sessionrestore_no_auto_restore)', 'Start-up (sessionrestore_no_auto_restore)', 'sessionrestore_no_auto_restore'],
     ['start-up-(ts_paint)', 'Start-Up (ts_paint)', 'ts_paint'],
+    ['sessionrestore_many_windows', 'Sessionrestore many windows', 'sessionrestore_many_windows'],
```
Comment (davehunt) [inline src/fission/config.js:36]:
```suggestion
    ['start-up-(sessionrestore_many_windows)', 'Start-up (sessionrestore_many_windows)', 'sessionrestore_many_windows'],
```

File: src/fission/config.js [hunk 2]
```diff
@@ -33,6 +33,9 @@ const PERFORMANCE_COMBO_TABLE = {
...
+    ['startup_about_home_paint', 'Startup about home paint', 'startup_about_home_paint'],
```
Comment (davehunt) [inline src/fission/config.js:37]:
```suggestion
    ['start-up-(about_home_paint)', 'Start-up (about_home_paint)', 'startup_about_home_paint'],
```

File: src/fission/config.js [hunk 3]
```diff
@@ -33,6 +33,9 @@ const PERFORMANCE_COMBO_TABLE = {
     ['start-up-(sessionrestore)', 'Start-up (sessionrestore)', 'sessionrestore'],
     ['start-up-(sessionrestore_no_auto_restore)', 'Start-up (sessionrestore_no_auto_restore)', 'sessionrestore_no_auto_restore'],
     ['start-up-(ts_paint)', 'Start-Up (ts_paint)', 'ts_paint'],
```
Comment (davehunt) [inline src/fission/config.js:35]:
```suggestion
    ['start-up-(ts_paint)', 'Start-up (ts_paint)', 'ts_paint'],
```

File: src/fission/config.js [hunk 4]
```diff
@@ -33,6 +33,9 @@ const PERFORMANCE_COMBO_TABLE = {
...
+    ['sessionrestore_many_windows', 'Sessionrestore many windows', 'sessionrestore_many_windows'],
+    ['startup_about_home_paint', 'Startup about home paint', 'startup_about_home_paint'],
+    ['startup_about_home_paint_realworld_webextensions', 'Startup about home paint realworld webextensions', 'startup_about_home_paint_realworld_webextensions'],
```
Comment (davehunt) [inline src/fission/config.js:38]:
```suggestion
    ['start-up-(about_home_paint_realworld_webextensions)', 'Start-up (about_home_paint_realworld_webextensions)', 'startup_about_home_paint_realworld_webextensions'],
```

**Review by davehunt — CHANGES_REQUESTED** [review]
The code changes look good, but I notice that the deploy review is failing to show any data on the Fission page. I've also added some suggestions to make the names/labels more consistent.

**Review by davehunt — APPROVED** [review]

**Pull request merged**